samples/subsys/audio/sof/prj.conf

```diff
@@ -1,4 +1,6 @@
+# See https://docs.zephyrproject.org/latest/guides/modules.html
```

Comment:
what are you try to say here exactly?

Reply:
I'm sure it's obvious to you but I found very obscure that it is this  `CONFIG_SOF` line that adds the module source to the build, this took me a very long time to figure it out. Considering modules can come with their own Kconfig options, I expected module selection to be performed at a higher level.

It's also surprisingly un-greppable, unlike say CONFIG_MODULE_SOF. The lack of prefix or suffix could also cause collisions where one fairly generic module name matches a CONFIG_ in another.

So I think all this deserves a comment; any comment with "module" somewhere on the line.